db/citations.yml

```diff
@@ -7346,6 +7346,34 @@ liu;sr14:
   wwwemail: ''
   wwwpub: http://slapper.apam.columbia.edu/bib-eu9iifae/papers/liu_sr14.pdf
   year: '2014'
+llan;RobustnessTestSpacegroupMining22:
+  ackno: This work in the Billinge group was supported by the U.S. National Science
+    Foundation through grant DMREF-1922234 and CCF-1704833.
+  author:
+    - Ling Lan
+    - Chia-Hao Liu
+    - Qiang Du
+    - Simon J. L. Billinge
+  doi: ''
+  entrytype: short communication
+  facility: ''
+  grant: dmref19
+  journal: Journal of Applied Crystallography
+  month: feb
+  nb: ''
+  note: arXiv:2202.12335 [cond-mat.mtrl-sci]
+  number: ''
+  optannote: ''
+  optwwwlanl: ''
+  pages: ''
+  supplementary_info_urls: https://gitlab.thebillingegroup.com/software/20ll_sgcnnrobusttest
+  title: Robustness Test of the spacegroupMining Space Group Determination Model for
```

Comment:
we need tags and synopsis.  Look at Tim's paper for examples (not sure if it has them but if it does they will be a good start).  tags are for building reading lists based on the tags.  Synopsis is a short description of the main result that would be appropriate for a web-site, for example.